src/Microsoft.ML.TensorFlow/Microsoft.ML.TensorFlow.csproj

```diff
@@ -10,7 +10,7 @@
   <ItemGroup>
     <PackageReference Include="System.IO.FileSystem.AccessControl" Version="$(SystemIOFileSystemAccessControl)" />
     <PackageReference Include="System.Security.Principal.Windows" Version="$(SystemSecurityPrincipalWindows)" />
-    <PackageReference Include="TensorFlow.NET" Version="0.10.10" />
+    <PackageReference Include="TensorFlow.NET" Version="0.11.3" />
```

Comment:
This line can actually be removed. Since this project references `Microsoft.ML.Dnn`, and `Microsoft.ML.Dnn` references TF.NET, we don't need to say that this project references TF.NET. That way one duplicated version number can be removed.